playbooks/resize_disk.yaml

```diff
@@ -0,0 +1,31 @@
+---
+- hosts: all
+  become: true
+  tasks:
+    - name: Install cloud-utils-growpart package
+      package:
+        name: cloud-utils-growpart
+
+    - name: Install e2fsprogs package
+      package:
+        name: e2fsprogs
+      when: ansible_distribution_major_version == '7'
+
+    - name: Extend partition
+      shell: growpart /dev/vda 1
```

Comment:
Isn't command safer than shell? I thought ansible-lint complained about that

Reply:
Generally yes, but depends on the use case. I had to go with shell here due to command erroring out. I'd have to dig up the error.